package.json

```diff
@@ -14,7 +14,7 @@
 	},
 	"dependencies": {
 		"@discordjs/opus": "^0.3.2",
-		"discord.js": "^12.3.1",
+		"discord.js": "github:discordjs/discord.js#af00ec8970e77ea8a0afd21571eeeef9c554e1ec",
```

Comment:
Probably have to keep an eye out for the next release so we can change this, everything else looks great!